src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Media_Animation/Given_DoubleAnimation.cs

```diff
@@ -162,5 +165,68 @@ public async void When_RepeatForever_ShouldLoop_AsdAsd()
 			Assert.AreEqual(10d, averageIncrement, 1.5, "an rough average of increment (exluding the drop) of 10 (+-15% error margin)");
 			Assert.IsTrue(incrementSizes.Count(x => x > 3) > 8, $"at least 10 (-2 error margin: might miss first and/or last) sets of continuous increments that size of 4 (+-1 error margin: sliding slot): {string.Join(",", incrementSizes)}");
 		}
+
+		[TestMethod]
+		[RunsOnUIThread]
+#if __ANDROID__
+		[Ignore("The native property (translate.View.Y) is being animated here instead of the dependency property (translate.Y).")]
```

Comment:
It seems that you added some code specific to Android to handle that case, is this `[Ignore]` still relevant ? If yes, could we add another test that will validate that platform specific code?